.github/ci.yaml

```diff
@@ -0,0 +1,18 @@
+name: Build code
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Lint
+      uses: make fix lint
+      with:
```

Comment:
These parameters are for the `golangci/golangci-lint-action` GItHub action. You don't need them here.